packages/trpc/src/mutation-event/comment-handler.ts

```diff
@@ -0,0 +1,213 @@
+import { getTextFromRteDoc, JSONContent } from '@chirpy-dev/utils';
+import { Comment } from '@prisma/client';
+import { NextApiResponse } from 'next';
+
+import { prisma } from '../common/db-client';
+import { revalidateCommentWidget } from '../common/revalidate';
+import { sendNotification } from '../notification/send';
+import { NotificationPayload } from '../notification/types';
+import {
+  createOneNotificationMessage,
+  deleteNotificationMessage,
+  getAuthorByCommentId,
+  getUserByPk,
+} from './utilities';
+
+export type EventPayload = {
+  op: 'INSERT' | 'DELETE';
+  comment: Pick<Comment, 'id' | 'content' | 'userId' | 'parentId'>;
+};
+
+/**
+ * Handle INSERT/DELETE comment event: send web notification, trigger revalidation of the comment page.
+ */
+export async function handleCommentEvent(
+  event: EventPayload,
+  res: NextApiResponse,
+  authUserId: string,
+): Promise<void> {
+  const promises = [];
+  const notificationPayloads: (NotificationPayload & { contextId: string })[] =
+    [];
+  if (event.op === 'INSERT') {
+    const commentId = event.comment.id;
+    const body = getTextFromRteDoc(event.comment.content as JSONContent);
```

Comment:
Remove `as JSONContent`

Reply:
<img width="877" alt="image" src="https://user-images.githubusercontent.com/7880675/204274673-92bfa4fb-fbd7-4205-864b-54c08929a9e1.png">
Have to keep it